gladier/__init__.py

```diff
@@ -2,3 +2,9 @@
 
 # https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library  # noqa
 logging.getLogger("gladier").addHandler(logging.NullHandler())
+
+from .client import GladierClient
+from .defaults import GladierDefaults as GladierFunction
+from .defaults import GladierDefaults
```

Comment:
Can just pick one of these? I think multiple names for things is going to cause confusion. I do think "GladierDefaults" isn't a great name and should be changed to something else. How about "GladierBaseTool"?

Reply:
GladierBaseTool is better. I could not remember the name you gave it that is why I duplicated it. I would also change the .default to .tool. comments?

Reply:
Yeah, I think we should change them all.